tests/core/operations/test_spatialdata_operations.py

```diff
@@ -284,6 +291,32 @@ def test_concatenate_sdatas(full_sdata: SpatialData) -> None:
     assert len(list(concatenated.gen_elements())) == 3
 
 
+@pytest.mark.parametrize("concatenate_tables", [True, False])
+@pytest.mark.parametrize("obs_names_make_unique", [True, False])
+def test_concatenate_sdatas_from_iterable(concatenate_tables: bool, obs_names_make_unique: bool) -> None:
```

Comment:
I'd probably add a test case that concatenating an iterable of length 1 still works and returns the identity... it's a commonly overlooked edge case.